src/consensus/tx_verify.cpp

```diff
@@ -83,8 +83,7 @@ std::pair<int, int64_t> CalculateSequenceLocks(const CTransaction &tx,
     // tx.nVersion is signed integer so requires cast to unsigned otherwise
     // we would be doing a signed comparison and half the range of nVersion
     // wouldn't support BIP 68.
-    bool fEnforceBIP68 = static_cast<uint32_t>(tx.nVersion) >= 2 &&
-                         flags & LOCKTIME_VERIFY_SEQUENCE;
+    bool fEnforceBIP68 = static_cast<uint32_t>(tx.nVersion) >= 2;
```

Comment:
Should we set this == 2? There aren't anything else right now, and we probably want to restrict it in the future. Though, I think there is a check for nVersion == 2 now.

On BCH this was sort of required because there were old transactions with the version set oddly.